core/src/main/resources/hudson/model/AllView/noJob.jelly

```diff
@@ -23,30 +23,47 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<div>
-  <!--
-    This is a good place to add "welcome" materials, like pointers to the website and etc
-  -->
-  <h1>${%Welcome to Jenkins!}</h1>
-
-  <j:choose xmlns:j="jelly:core">
-    <!-- Only show the create new jobs link to those users that have permission to use it. -->
-    <j:getStatic var="permission" className="hudson.model.Item" field="CREATE"/>
-    <j:when test="${h.hasPermission(permission)}">
-      <div class="call-to-action">${%newJob}</div>
-    </j:when>
-    <j:otherwise>
-      <!-- Offer anonymous users the chance to login, then they can create jobs. -->
-      <j:if test="${h.isAnonymous()}">
-        <div class="call-to-action">
-        ${%login(rootURL, app.securityRealm.loginUrl, request.requestURI)}
-        <!-- Also offer them the opportunity to sign-up for an account, if this facility is available. -->
-        <j:if test="${app.securityRealm.allowsSignup()}">
-          ${%signup}
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+  <l:js src="jsbundles/add-item.js" />
+  <l:css src="jsbundles/jenkins-widgets.css" />  
+  <l:css src="css/icons/icomoon.css" />  
+  
+  <div class="welcome-message max-read-width read">
+    <i class="icon-service-o watermark"></i>
+    <!--
+      This is a good place to add "welcome" materials, like pointers to the website and etc
+    -->
+    <h1>${%Welcome to Jenkins!}</h1>
+    <p>This is a Jenkins automation and build job server. You currently have no jobs or tasks configured to run on this server. 
```

Comment:
@gusreiber My point is, it's only available to be localized when placed in `${%…}`. See https://wiki.jenkins-ci.org/display/JENKINS/Internationalization#Internationalization-Markingmessagesinjellyfiles